egs2/fleurs/asr1/run.sh

```diff
@@ -43,13 +47,36 @@ if [[ "all" == *"${lang}"* ]]; then
       --feats_type raw \
       --speed_perturb_factors "0.9 1.0 1.1" \
       --asr_config "${multilingual_asr_config}" \
+      --train_set "${train_set}" \
+      --valid_set "${train_dev}" \
+      --test_sets "${test_set}" \
+      --bpe_train_text "data/${train_set}/text" \
+      --lm_train_text "data/${train_set}/text" "$@" 
+
+  # create auxillary labels once pre-processing is finished
+  python local/create_lids.py
```

Comment:
This looks odd, but the auxiliary labels need to be created after the speech perturbation stage so that the speech perturbation samples also have auxiliary labels

Reply:
I would suggest add the process into TEMPLATE instead of here. We can always make it optional to the template anyway.